install/updates/froxlor/0.9/update_0.9.inc.php

```diff
@@ -3298,7 +3298,7 @@
 	Settings::AddNew("system.ssl_protocols", "TLSv1,TLSv1.1,TLSv1.2");
 	
 	showUpdateStep("Adding ssl_protocols setting");
-	$update_ssl_protocols = isset($_POST['update_ssl_protocols']) ? $_POST['update_ssl_protocols'] : "TLSv1,TLSv1.1,TLSv1.2";
+	$update_ssl_protocols = isset($_POST['update_ssl_protocols']) ? implode(',', $_POST['update_ssl_protocols']) : "TLSv1,TLSv1.1,TLSv1.2";
 	Database::query("INSERT INTO `" . TABLE_PANEL_SETTINGS . "` SET `settinggroup` = 'system', `varname` = 'ssl_protocols', `value` = '" . $update_ssl_protocols ."'");
```

Comment:
as said many times, this should be
`Settings::AddNew("system.ssl_protocols", $update_ssl_protocols);`